lexers/t/teraterm.go

```diff
@@ -0,0 +1,230 @@
+package t
+
+import (
+	"regexp"
+	"strings"
+
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+var teraTermAnalyserCommandRe = regexp.MustCompile(`(?i)\b(` + strings.Join([]string{
+	"basename",
+	"beep",
+	"bplusrecv",
+	"bplussend",
+	"break",
+	"bringupbox",
+	"callmenu", // 'call' is handled separately.
```

Comment:
The comment refers to the place between `bringupbox` and `callmenu`.
```suggestion
	"bringupbox",
	// 'call' is handled separately.
	"callmenu",
```